scripts/__tests__/build-blog-data-integration.test.js

```
@@ -0,0 +1,432 @@
+
+        const outputPath = path.join(tempDir, 'lib', 'blog-data.json')
+        const blogData = JSON.parse(fs.readFileSync(outputPath, 'utf8'))
+
+        const post = blogData.posts[0]
+
+        // Verify reading time is calculated and is reasonable
+        expect(post.readTime).toBeDefined()
+        expect(post.readTime).toMatch(/\d+ min read/)
+
+        // Extract the number of minutes
+        const minutes = parseInt(post.readTime.match(/(\d+) min/)[1])
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)